frontend/src/components/system-upgrade/index.vue

```diff
@@ -158,26 +111,6 @@ const onLoadUpgradeInfo = async () => {
         });
 };
 
-const changeOption = async () => {
-    const res = await loadReleaseNotes(upgradeVersion.value);
-    upgradeInfo.value.releaseNote = res.data;
-};
-
-const onUpgrade = async () => {
-    ElMessageBox.confirm(i18n.global.t('setting.upgradeHelper', i18n.global.t('commons.button.upgrade')), {
-        confirmButtonText: i18n.global.t('commons.button.confirm'),
-        cancelButtonText: i18n.global.t('commons.button.cancel'),
-        type: 'info',
-    }).then(async () => {
-        globalStore.isLoading = true;
-        await upgrade(upgradeVersion.value);
-        globalStore.isOnRestart = true;
-        drawerVisible.value = false;
-        MsgSuccess(i18n.global.t('commons.msg.operationSuccess'));
-        search();
-    });
-};
-
 onMounted(() => {
     isProductPro.value = globalStore.isProductPro;
     search();
```

Comment:
There are no syntax errors or inconsistencies here. You should continue optimizing this template to ensure it's clear and concise without using unnecessary features. Here's a suggested improvement:

```html
<!-- Importing styles directly -->
<link rel="stylesheet" href="path/to/md-editor-v3/lib/style.css">

<Upgrade></Upgrade>

<div id="system-update-container">
  <!-- Other components... -->

</div>
```

This way, you avoid adding the MD editor CSS in each component file which could potentially lead to additional bundling.

For more improvements:
* Split long strings into smaller chunks when displaying them in templates.
* Use dynamic imports rather than importing files inline at runtime.
* Implement consistent spacing and indentation for improved readability.

Feel free to ask about specific optimizations based on these general principles. For example: "Would you suggest I use camelCase over camelCaps naming for property names?"